deploy/examples/common.yaml

```diff
@@ -232,6 +232,1071 @@ rules:
     resources: ["tokenreviews"]
     verbs: ["create"]
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: release-name-ceph-csi-operator-cephconnection-viewer-role
+  labels:
+    app.kubernetes.io/name: ceph-csi-operator
+    app.kubernetes.io/instance: release-name
+    app.kubernetes.io/version: "0.2.0"
```

Comment:
strange that the version of all these resources is 0.2.0, but the helm dependency is for 0.3.1, perhaps an issue with the helm chart?

Reply:
i think its based on the appVersion field in the Charts, there are two fields in the charts, one is version and another is appVersion, where appVersion is not a mandatory field like version, but it represents the version of the driver. i will get it fixed in the ceph-csi-operator charts.